docs/guides/python-tests.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: "Python tests"
+---
+
+## Python in dbt
+
+In dbt v1.3.0, Python models were introduced in order to solve use cases that you can't solve with SQL.
+Using Python allows you to perform analyses using tools available in the open source Python ecosystem, including
+state-of-the-art packages for data science and statistics.
+Before, you would have needed separate infrastructure and orchestration to run Python.
+
+For more information, review
+the [Python models documentation](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/python-models).
+
+## What about tests?
+
+Just like with models, we'd also like to be able to test our data using those powerful tools and the convenient Python
+runtime.
+Certain problems such as validating JSONs or comparing tables become tremendously easier to solve.
+Currently, dbt doesn't provide an out-of-the-box mechanism to run tests using Python. Elementary now provides the
+ability to do so using dbt's building blocks.
+
+## Elementary Python tests
+
+Elementary introduces a generic dbt test called `elementary.python` that allows users to run Python code and validate
+their data.
```

Comment:
Can you just add:
"This is a beta feature, introduced in version 0.5.2 of Elementary [dbt package](https://github.com/elementary-data/dbt-data-reliability)."